lib/transport/v8-transport-helper.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Shareable } from '../memory/shareable';
+
+/// <summary>
+/// SharedPtrWrap of SerializedData.
+/// </summary>
+export interface SerializedData extends Shareable {
```

Comment:
SharedBuiltInType?

Reply:
This is the js object generated for all values passed to serializeValue, so keep it as serializedData

---
In reply to: [154741218](https://github.com/Microsoft/napajs/pull/137#discussion_r154741218) [](ancestors = 154741218)